runscripts/context_gating.sh

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-python run_context_gating.py 'seed=range(0,5)' '+experiment=glob(*)' '+algorithm=sac' '+environment=pendulum' 'contexts.context_feature_args=[],[g],[max_speed],[l],[m],[dt]' 'carl.state_context_features=null,${contexts.context_feature_args}' 'carl.gaussian_noise_std_percentage=0.4' --multirun
\ No newline at end of file
+python ~/git/CARL/experiments/context_gating/run_context_gating.py 'seed=range(0,5)' '+experiment=glob(*)' '+algorithm=ddpg' '+environment=pendulum' 'contexts.context_feature_args=[max_speed],[g],[max_speed],[l],[m],[dt]' 'carl.state_context_features=null,${contexts.context_feature_args}' 'carl.gaussian_noise_std_percentage=0.4' --multirun
```

Comment:
- [ ] change to original path (just `run_context_gating.py`)
- [ ] write `[]` instead of first `[max_speed]`